src/com/vladsch/clionarduinoplugin/serial/SerialMonitorToolWindow.java

```diff
@@ -47,20 +47,22 @@ public class SerialMonitorToolWindow {
 
     public SerialMonitorToolWindow(Project project) {
         this.project = project;
-        toolWindow = ToolWindowManager.getInstance(project).registerToolWindow(TOOL_WINDOW_ID, false, ToolWindowAnchor.BOTTOM, project, true);
-        toolWindow.setIcon(PluginIcons.serial_port_tool);
-
-        //JPanel mainPanel = new JPanel(new BorderLayout());
-        //mainPanel.setBorder(BorderFactory.createEmptyBorder(0, 0, 0, 0));
-        SimpleToolWindowPanel mainPanel = new SimpleToolWindowPanel(false, true);
-
-        mySerialMonitorPanel = new SerialMonitorPanel(project);
-        mainPanel.add(mySerialMonitorPanel.getComponent());
-
-        ContentFactory contentFactory = ContentFactory.SERVICE.getInstance();
-        Content content = contentFactory.createContent(mainPanel, Bundle.message("toolwindow.serial-monitor.title"), false);
-        myToolWindowContentMap.put(SERIAL_MONITOR_ID, content);
-        toolWindow.getContentManager().addContent(content);
+        toolWindowManager.invokeLater(() -> {
```

Comment:
I changed it this way and it worked.

```suggestion
        ToolWindowManager.getInstance(project).invokeLater(() -> {
```

https://github.com/eiryu/CLionArduinoPlugin/pull/1/commits/a7c5854b4fd077cc39d8839d2c318da4f31e5ac3

Reply:
<img width="1385" alt="image" src="https://user-images.githubusercontent.com/926934/186258090-bd7054b8-f7d5-4583-aae9-5c101fac61d7.png">